src/plugins/health.ts

```diff
@@ -5,29 +5,29 @@ export interface QueryOptions {
    * Start date from which to get data
    */
   startDate: Date;
-  
+
```

Comment:
Please resubmit PR without  any modifications to this file `health.ts`. You need to merge your branch correctly with master branch.

Alternatively, you can copy your edits into a new branch based on `master`.